back/app/serializers/web_api/v1/folder_serializer.rb

```diff
@@ -3,6 +3,18 @@
 class WebApi::V1::FolderSerializer < WebApi::V1::BaseSerializer
   attributes :title_multiloc, :description_preview_multiloc, :slug, :created_at, :updated_at
 
+  attribute :comments_count do |object|
```

Comment:
Feels like these numbers should be in a service somewhere, but I don't think there's anything truly appropriate at the moment - doesn't quite fit with the participants service - so I think this is fine.

Reply:
Yes, I thought exactly the same 😅